### PR TITLE
HTML5 audio and video backends

### PR DESCRIPTION
**Pull request by thijstriemstra**
refs #57

### CODE REVIEW

File: embed_video/backends.py [hunk 1]
```diff
@@ -280,6 +285,71 @@ def set_options(self, options):
...
 
+class Html5Backend(VideoBackend):
+    """
+    Backend for HTML5 media.
+    """
+    def get_url(self):
+        """
+        Returns URL.
+        """
+        return mark_safe(self._url)
```
Comment (yetty) [inline embed_video/backends.py]: Why is there called `mark_safe`?

**Comment by thijstriemstra** [comment]
@yetty could you please take a look?

**Comment by SalahAdDin** [comment]
@yetty 👍

**Comment by yetty** [comment]
@thijstriemstra I am sorry, I have absolutly no time to finish this. [Wouldn't you like to become a maintainer](https://github.com/yetty/django-embed-video/issues/59)?

**Comment by SalahAdDin** [comment]
I can do a merge pr, but i don't know how to update pypi packages and add new features.

**Comment by SalahAdDin** [comment]
@yetty Please, merge this PR.

**Pull request closed without merging**